Content.Shared/Pinpointer/SharedNavMapSystem.cs

```diff
@@ -237,7 +269,13 @@ public void ApplyToFullState(IComponentState fullState)
             }
 
             foreach (var (chunk, data) in Chunks)
-                state.Chunks[chunk] = new(data);
+            {
+                for (var i = 0; i < NavMapComponent.Categories; i++)
+                {
+                    var chunkData = data[i];
+                    state.Chunks[chunk][i] = chunkData == null ? chunkData : new(chunkData);
```

Comment:
Is this supposed to be flipped? If `chunkData` is `null` it will pass `null` to `state.Chunks[chunk][i]`.

Reply:
I think its intentional? if its null it stays null, but if it is not null it creates a clone of the data so that they don't both point to the same list